packages/navstar/extra/dcos-navstar.service

```diff
@@ -21,4 +21,7 @@ ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-navstar
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-minuteman
 ExecStartPre=/usr/bin/mkdir -p /var/lib/dcos/navstar/mnesia
 ExecStartPre=/usr/bin/mkdir -p /var/lib/dcos/navstar/lashup
+ExecStartPre=/usr/bin/env modprobe dummy
+ExecStartPre=-/usr/bin/env ip link add minuteman type dummy
```

Comment:
@asridharan @lingmann filed a jira for this, https://mesosphere.atlassian.net/browse/DCOS-13698
i think it can be handled in a separate pull request.  we don't have a "way to do this" yet afaik.  so it will take a little bit of thought to implement